README.md

```diff
@@ -49,18 +49,27 @@ env:
 jobs:
   deploy_to_production:
     runs-on: ubuntu-latest
-    name: deploy to production
+    name: deploy to production with artifact
     steps:
       - name: Checkout Github Action
         uses: actions/checkout@master
+        
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: maven build, clean
+        run: |
+          mvn clean package -DskipTests
```

Comment:
consider remove `-DskipTests`. When run in GitHub Actions, it's common to run all the tests to ensure the correctness.